content/source/docs/enterprise/before-installing/index.html.md

```diff
@@ -128,29 +126,117 @@ See [Network Requirements](./network-requirements.html) for details.
 
 <a id="software-requirements"></a>
 
-### Software Requirements (Standalone Deployment)
-
-Every Terraform Enterprise installation requires the following:
-Enterprise:
-
-- A 64-bit architecture Linux-based operating system
-- Linux Kernel 3.10 or greater
-- Docker Engine 17.06.2-ce to 20.10:
-  - Docker Engine 18.01.0-ce or greater requires Replicated 2.32.0
-    or greater
-  - Support for [Alpine 3.14](https://wiki.alpinelinux.org/wiki/Release_Notes_for_Alpine_3.14.0#faccessat2)
-    Docker images:
-    - Docker Engine 17.06.2-ce to 20.09 requires runc v1.0.0-rc93
-    - Docker Engine 20.10 or greater requires libseccomp 2.4.4
-  - In Online mode, the installer will install Docker Engine
-    automatically
-  - In Airgapped mode, you must install Docker Engine before running
-    the installer
-
-Some operating systems also have additional requirements:
-
-- [RedHat Enterprise Linux (RHEL) Requirements](./rhel-requirements.html)
-- [CentOS Requirements](./centos-requirements.html)
+### Docker Engine Requirements
+
+Terraform Enterprise requires **at least one** of the following Docker Engine configurations, in order of preference:
+
+  1. Docker Engine 17.06.2-ce, 18.09.x, 19.03.x, or 20.10.x with `runc` v1.0.0-rc93 or greater.
+  2. Docker Engine 20.10.x with `libseccomp` 2.4.4 or greater.
+  3. Docker Engine 17.06.2-ce, 18.09.x, 19.03.x, or 20.10.x using a modified `libseccomp` profile.
+
```

Comment:
Should we include some information here about the online installer installing Docker Engine for you, but not verifying the `runc` et al requirements?

Similarly, should we include information here about the airgap installation needing to install Docker Engine prior to the installation process being executed?

If so, what should that wording look like?

Reply:
For the first point:
```suggestion

-> **Note:** The online installer is not capable of verifying the Docker Engine configuration requirements.

```

For the second point, the instructions as proposed already include the installation of Docker Engine, so perhaps we can modify them to note that step is only necessary for airgap installations.

Reply:
Yeah, maybe something like "New, online installations of Terraform Enterprise will have the appropriate version of Docker installed automatically. Airgap installations, and upgrades (online and airgap) will require manual intervention." ??